collector/src/main/java/com/bitmovin/analytics/CollectorConfig.java

```diff
@@ -53,4 +56,16 @@ public String getBackendUrl() {
     public void setBackendUrl(String backendUrl) {
         this.backendUrl = backendUrl;
     }
+
+    /**
+     * Returns true if the stream is live.
```

Comment:
```suggestion
     * Returns true if the stream is marked as live before stream metadata is available.
```

Reply:
Think also the `@return` should not be empty: https://stackoverflow.com/questions/10088311/javadoc-return-tag-comment-duplication-necessary